src/script/countries/ROU.coffee

```diff
@@ -14,6 +14,14 @@ class Romania
 
 
 	specialRules: (withoutCountryCode, withoutNDC, ndc) =>
+		if withoutCountryCode[0] is '0'
+			# Sometimes clients may type their phones with a leading zero
+			# we should strip that
+			withoutNDC = withoutCountryCode.slice(1)
+		
+		if withoutCountryCode.slice(0,2) is 40
```

Comment:
shouldn't `40` be a string?

Reply:
Another mistery I couldn't solve, I tested as a string and it didn't work, then as a number and it did. 🤷

Reply:
Nice! In the future, we could use some variables instead of these magic numbers.